springfield/cms/templates/cms/includes/menus/features.html

```diff
@@ -0,0 +1,86 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public License,
```

Comment:
I've noticed that on the HTML files you added in this branch, when the pre-commit hook runs, the format of the license doesn't match what it's looking for, so it tries to add it again. If you edit the html files, stage them, then run `pre-commit run`, you'll see what I mean. It might be good to update those now so it doesn't cause issues with the precommit hooks in the future.